include.js

```diff
@@ -4143,6 +4143,13 @@ let cards = [
     imageLink: './Art/AnnaWijetunga/orange.gif', 
     author: 'Anna',
     githubLink: 'https://github.com/AnnaWijetunga' 
+  },
+  {
+    artName: 'Animator', // change this to the name of your artwork
+    pageLink: './Art/Animator/index2.html', // change this
+    imageLink: './Art/Animator/triangle.gif', // change this
+    author: 'ESAM', // use your name
+    githubLink: 'https://github.com/ESAMHADI' // change this
   }
```

Comment:
# Incorrect Include.js 
You need to provide:
- a valid folder name - named after your github user name
- valid paths for : pageLink. imageLink - Animator is not the name of your folder 
  - a typo is identified ( the folder is animatron and not animator )
  - the folder name does not correspond to your user name as mentioned in the first bullet.
- please remove comments - those are there to help but need to be removed.